### DISCUSSION TITLE
Customizable iOS app icons for light and dark mode

### DISCUSSION THREAD

**Discussion by farmio** [community/community#139257] [Mobile]
It would be awesome if we could select different custom icons for iOS 18 light and dark mode 😃

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.178.0 (141531614)
Device: iPhone 12 mini (iOS 18.0)
</details>

**Comment by HenSquared** [comment]
@farmio, nice idea. :) 

I'll make an issue internally to track this. Thanks for the feedback.

**Reply by farmio** [reply]
Nice 👍 
I'd love to have my "Frozen Melon" during day and "Berry Smoothie" at nighttime 😃
